pkg/network/ebpf/c/prebuilt/shared-libraries.c

```diff
@@ -0,0 +1,12 @@
+#include "kconfig.h"
+#include "bpf_tracing.h"
+#include "bpf_telemetry.h"
+#include "bpf_builtins.h"
+
+#include <uapi/linux/ptrace.h>
+
+#include "shared-libraries/types.h"
+#include "shared-libraries/maps.h"
+#include "shared-libraries/probes.h"
```

Comment:
I'd add a comment 
"All probes are shared among prebuilt and runtime, and can be found here"